meson.build

```diff
@@ -43,7 +43,10 @@ add_project_arguments(c_args, language: 'c')
 
 cc = meson.get_compiler('c')
 
-librt = cc.find_library('rt', required: true)
+librt = []
+if host_machine.system() == 'linux'
```

Comment:
What about FreeBSD?

It would be better to make it an optional dependency. If it's not found, it's probably not needed.

Reply:
done.